src/cctag/utils/Defines.hpp

```diff
@@ -64,6 +64,12 @@
 #include <cstddef>
 namespace std {
 #ifdef _MSC_VER
+#if defined(_WIN64)
+	 typedef __int64 LONG_PTR;
+#else
```

Comment:
@zvrba @fabiencastan 
Actually I see that there is a header `<basetsd.h>` that defines SSIZE_T as
```c++
typedef LONG_PTR SSIZE_T;
```
so we can use that, something in the likes of
```c++
#ifdef _MSC_VER
#include <basetsd.h>
using ssize_t = SSIZE_T;
#else
...
```
What do you think?

Reply:
1) It is "UB" to add members to `std` namespace. 2) `ssize_t` is POSIX only, not ANSI C. 3) Replace `ssize_t` with `intptr_t`.